model_compression_toolkit/core/pytorch/reader/reader.py

```diff
@@ -17,16 +17,85 @@
 import logging
 from typing import Callable, Dict
 
-import numpy as np
 import torch
-from torch.fx import symbolic_trace
+from torch.fx import GraphModule, Tracer
 from torch.fx.passes.shape_prop import ShapeProp
 
 from model_compression_toolkit.core.common import Graph
 from model_compression_toolkit.core.pytorch.reader.graph_builders import edges_builder, nodes_builder
 from model_compression_toolkit.core.pytorch.utils import set_model
 
 
+class LeafModulesTracer(Tracer):
+    """
+    LeafModulesTracer overrides is_leaf_module method to make symbolic trace
+    model contains layers, using inputs variables to control flow.
+    """
```

Comment:
LeafModulesTracer overrides the 'is_leaf_module' method. 

I don't understand the rest of the sentence:
" to make symbolic trace model contains layers, using inputs variables to control flow."